<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaticMaps</class>
 <widget class="QMainWindow" name="StaticMaps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="map">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>461</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="coord1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="coord2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>660</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Широта</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Долгота</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Схема</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>140</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Спутник</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Гибрид</string>
    </property>
   </widget>
   <widget class="QPushButton" name="on_b">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить вид карты</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
